pnpm-lock.yaml

```diff
@@ -164,6 +164,9 @@ importers:
       dayjs:
         specifier: ^1.11.13
         version: 1.11.13
+      fuse.js:
```

Comment:
Would recommend removing `fuse.js` from `apps/web` and running `pnpm install` again from the project root to revert this. Or you could just revert the `package.json` in `apps/web` and revert this `pnpm-lock.yaml` file.

Reply:
Removed install in apps/web, moved it to the packages in the api package.